tensorboard/plugins/hparams/tf_hparams_sessions_pane/tf-hparams-sessions-pane.ts

```diff
@@ -29,18 +29,33 @@ import '../tf_hparams_table_view/tf-hparams-table-view';
  */
 @customElement('tf-hparams-sessions-pane')
 class TfHparamsSessionsPane extends PolymerElement {
+  constructor() {
+    super();
+    // If window['ga'] is defined, use it, otherwise any logging calls will be
+    // no-ops.
+    // @ts-ignore
+    this.ga = window['ga'] || function () {};
```

Comment:
It would be nice to wrap this in some utility function? It would set this up in a way that can be shared.
What are the circumstances which can cause `window['ga']` to be `undefined`? When the script hasn't loaded?